### PR TITLE
fix the repetitive word

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: compiler/rustc_trait_selection/src/solve/eval_ctxt.rs [hunk 1]
```diff
@@ -537,7 +537,7 @@ impl<'a, 'tcx> EvalCtxt<'a, 'tcx> {
 
     /// Iterate over all added goals: returning `Ok(Some(_))` in case we can stop rerunning.
     ///
-    /// Goals for the next step get directly added the the nested goals of the `EvalCtxt`.
+    /// Goals for the next step get directly added the nested goals of the `EvalCtxt`.
```
Comment (BoxyUwU) [inline compiler/rustc_trait_selection/src/solve/eval_ctxt.rs:540]:
```suggestion
    /// Goals for the next step get directly added to the nested goals of the `EvalCtxt`.
```

File: src/ci/docker/host-x86_64/dist-mips-linux/patches/glibc/2.23/0002-MIPS-SPARC-more-fixes-to-the-vfork-aliases-in-libpth.patch [hunk 2]
```diff
@@ -8,7 +8,7 @@ Commit 43c29487 tried to fix the vfork aliases in libpthread.so on MIPS
 and SPARC, but failed to do it correctly, introducing an ABI change.
 
 This patch does the remaining changes needed to align the MIPS and SPARC
-vfork implementations with the other architectures. That way the the
+vfork implementations with the other architectures. That way the
```
Comment (klensy) [inline src/ci/docker/host-x86_64/dist-mips-linux/patches/glibc/2.23/0002-MIPS-SPARC-more-fixes-to-the-vfork-aliases-in-libpth.patch:11]: Don't think that `.patch`es should be changed.

**Comment by workingjubilee** [comment]
I mean, also
@bors r+

**Pull request merged**